scripts/validations.py

```diff
@@ -0,0 +1,125 @@
+import itertools
+import json
+import sys
+
+ACTION_CONTEXT = 'context'
+ACTION_COUNT = 'count'
+ACTIONS = (ACTION_CONTEXT, ACTION_COUNT)
+
+UNLISTED_ADDONS = set()
+
+
+def parse_validations(results):
+    return (json.loads(result) for result in results)
+
+
+def unlisted_validations(results, unlisted_addons=None):
+    if unlisted_addons is None:
+        unlisted_addons = UNLISTED_ADDONS
+    return (result
+            for result in results
+            if (not result['metadata'].get('listed', True)
+                or result['metadata'].get('id') in unlisted_addons))
```

Comment:
> 100% of validations with listed as True or None must have a matching unlisted add-on, because you're filtering through unlisted_validations before counting, so they'd be filtered out otherwise.

Good call, I didn't mean to include that. I removed that filter and these are the results:

```
((None, False, False), 2282)
((None, False, True), 878)
((None, True, False), 310)
((False, False, False), 223)
((False, False, True), 522)
((False, True, False), 23)
((True, False, False), 1508)
((True, False, True), 58)
((True, True, False), 86)